ReactiveCocoa/Swift/UnidirectionalBinding.swift

```diff
@@ -0,0 +1,313 @@
+import enum Result.NoError
+
+precedencegroup Binding {
```

Comment:
The name is not obvious. I think we should consistently add `Precedence` suffix.